app/assets/javascripts/constants/image_constants.es6.jsx.erb

```diff
@@ -1,11 +1,19 @@
 (() => {
   class ImageConstants {
-    
+
     get logos() {
       return {
         orange: "<%= asset_path("logo/logo_orange.png") %>"
       };
     }
+
+    get roles() {
+      return {
+        donor: "<%= asset_path("roles/donor.png") %>",
+        teacher: "<%= asset_path("roles/teacher.png") %>"
```

Comment:
isn't there a student.png too? I think you're using the same one for student/teacher rn